solutions/client-landing-zone/setters.yaml

```diff
@@ -104,12 +104,9 @@ data:
   # Deny sanctioned countries ingress traffic
   # https://cloud.google.com/firewall/docs/firewall-policies-rule-details#geo-location-object
   # https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements
-  # The list below is Google's default, modify as required
+  # The list below MUST BE reconfigured. If the value stays XX it will fail reconciliation with an error for unrecognized country.
   denied-sanctioned-countries: |
-    - "CU"
-    - "IR"
-    - "KP"
-    - "SY"
+    - "XX"
```

Comment:
good pattern for forced review of the sanction list! - to force an error via purposely busted list